tests/sympc/tensor/replicatedshare_tensor_test.py

```diff
@@ -150,11 +150,15 @@ def test_rst_distribute_reconstruct(get_clients, parties, security) -> None:
     session = Session(protocol=protocol, parties=parties)
     SessionManager.setup_mpc(session)
 
-    secret = 42.32
+    # Test with torch matrix
+    secret1 = torch.ones([10, 10]) * 5
```

Comment:
why not include zero and negative numbers  ? or a more mixed matrix?

Reply:
Sure.